stub/Makefile

```diff
@@ -1,15 +1,15 @@
 SHELL=/bin/bash
 
 clean:
-	cd ..; scripts/HootEnv.sh $(MAKE) -f Makefile.old clean
+	cd ..; scripts/HootEnv.sh $(MAKE) -f Makefile.hoot clean
 check:
-	cd ..; scripts/HootEnv.sh $(MAKE) -f Makefile.old check
+	cd ..; scripts/HootEnv.sh $(MAKE) -f Makefile.hoot check
 
 install:
 uninstall:
 
 all:
-	cd ..; scripts/HootEnv.sh $(MAKE) -f Makefile.old all
+	cd ..; scripts/HootEnv.sh $(MAKE) -f Makefile.hoot all
 
 default:
-	cd ..; scripts/HootEnv.sh $(MAKE) -f Makefile.old
+	cd ..; scripts/HootEnv.sh $(MAKE) -f Makefile.hoot
```

Comment:
I tried to remove this file and relocate these targets but that didn't work.  The reasons for this file are lost in history but removing it didn't work, so it got updated.